pkg/chartvalues/aws_operator_test.go

```diff
@@ -145,6 +154,11 @@ func Test_NewAWSOperator(t *testing.T) {
       Vault:
         Address: http://vault.default.svc.cluster.local:8200
     Guest:
+      Calico:
+        CIDR: 16
+        Subnet: "192.168.0.0"
+      Docker:
+        CIDR: "172.17.0.1/16"
```

Comment:
More necessary settings are put here.

Reply:
Sorry for my ignorance: Were these added recently? Or was a Docker CIDR always missing?

Reply:
I just saw the other PR, all good.